src/Http/Headers/src/HeaderNames.cs

```diff
@@ -65,7 +65,7 @@ public static class HeaderNames
         public static readonly string Authorization = "Authorization";
 
         /// <summary>Gets the <c>baggage</c> HTTP header name.</summary>
-        public static readonly string Baggage = "baggage";
+        public static readonly string Baggage = "Baggage";
```

Comment:
Why this change?

Reply:
Is to match the casing in KnownHeaders; otherwise it lowercases all the variable names, is case insensitive so doesn't really matter, but the other way has more code changes https://github.com/dotnet/aspnetcore/blob/7dea0cb6736bc8ea2b53e5a716b926e7a80a4430/src/Servers/Kestrel/shared/KnownHeaders.cs#L112